docs/input/docs/upgrading/1.x-to-2.x.md

```diff
@@ -0,0 +1,211 @@
+
+### Remove input parameters which are no longer required
+
+Input parameters for the [SetParameters](../fundamentals/set-parameters) have changed.  For example the `shouldRunGitVersion` parameter is now called `shouldCalculateVersion`.
+
+In addition, the default values of some parameters have changed, and may no longer need to be set.  For example, `shouldRunCodecov` is now enabled by default.
+
+### Change Cake.Recipe pre-processor directive to use latest version
+
+The entry pre-processor directive should be changed to the following:
+
+```csharp
+#load nuget:?package=Cake.Recipe&version=2.0.0
+```
+
+### Pin to updated version of Cake
+
+Cake.Recipe 1.x.x was pinned to run against version 0.32.1 of Cake.  In order to use Cake.Recipe 2.0.0 you will need to use at least 0.38.4.
+
+### Update bootstrapper to include --bootstrap command
+
+Cake.Recipe now includes the use of the .Net Global Tool Module.  As such, it will be necessary to first bootstrap this module, before executing the build.  The can be done as follows:
+
+```powershell
+Write-Host "Bootstrapping Cake"
+dotnet cake $SCRIPT_NAME --bootstrap
+if ($LASTEXITCODE -ne 0) { exit $LASTEXITCODE }
+
+Write-Host "Running Build"
+dotnet cake $SCRIPT_NAME @args
+if ($LASTEXITCODE -ne 0) { exit $LASTEXITCODE }
+```
+
+### Rename AppVeyor default target
+
+Due to the fact that Cake.Recipe is no longer directly coupled to AppVeyor, the entry target name has changed.  Instead of calling:
+
+```yml
+build_script:
+  - ps: .\build.ps1 -Target AppVeyor
+```
+
+you should call:
+
+```yml
+build_script:
+  - ps: .\build.ps1 --target=CI
+```
+
+### Update GitVersion.yml file with the required changes for using new version of GitVersion
+
+Cake.Recipe has switched from using 3.6.5 of GitVersion, to 5.0.1 of the .Net Framework version, and 5.3.7 of the .Net Global Tool version.  There have been a number of breaking changes in how GitVersion configuration works, specifically around how branches are configured.  If you have done any branch configuration in the GitVersion.yml file, you may have to update to the [latest standards](https://gitversion.net/docs/configuration#branch-configuration).
+
+### Ensure AppVeyor build image is using latest
+
+In order to execute correctly on AppVeyor, you will need to use the Visual Studio 2019 build image.  Change the image definition from:
+
+```yml
+image: Visual Studio 2017
+```
+
+to:
+
+```yml
+image: Visual Studio 2019
+```
+
+### Ensure that Environment.SetVariableNames() is called
+
+Previously, the call to [Environment.SetVariableNames](../fundamentals/set-variable-names) was not mandatory, and could be omitted from the recipe.cake file.  This is no longer the case, and it must be included.
+
+## Suggested Changes
+
+### Use embedded icon for NuGet package
+
+This guidance comes from the NuGet team directly.  Going forward, you should look to include an icon within the nupkg file.
```

Comment:
Suggestion from Grammarly:

> Some readers see phrases like **Going forward** as stiff or just too vague. Consider rephrasing.

I tend to agree here, but not sure what would be appropriate.
The tool suggest one of the following: `In the future`, `From now on` or `As we advance`.
Neither of those feel completely right (except maybe `From now on`).

Reply:
:+1: